metadata/requirements.txt

```diff
@@ -35,7 +35,7 @@ typing-extensions==3.7.4
 
 # A common package that holds the models deifnition and schemas that are used
 # accross different amundsen repositories.
-amundsen-common>=0.11.0
+amundsen-common>=0.10.0
```

Comment:
this must be at least 0.10.1 in order to work with lineage. 
0.10.1 introduced the `parent` field.

Reply:
yep it's just temporary as 0.11.0 is not yet published, agree it needs to be reverted before merging

Reply:
11 has been published 👍

Reply:
thanks, rebased!